k-distribution/include/builtin/domains.k

```diff
@@ -590,19 +589,31 @@ module ID-PROGRAM-PARSING
               | #UpperId                                             [notInRules, token, autoReject]
 endmodule
 
-module ID
-  imports STRING
+module ID-SYNTAX
+  imports INT-SYNTAX
+  imports STRING-SYNTAX
 
-  syntax Id
+  syntax Id [token]
```

Comment:
I'm not up to date with the latest changes in K5. Is the meaning of `token` changed or extended?

Reply:
This is being used to indicate that the abstract syntax of a sort contains KToken, even though it's not part of the parser.